RELEASE.md

```diff
@@ -1,3 +1,46 @@
+# Release 1.12.0
+
+## Major Features and Improvements
+* tf.data users can now use `tf.data.experimental.AUTOTUNE` for the num_parallel_calls argument of `tf.data.Dataset.interleave`, `tf.data.Dataset.map`, and `tf.data.experimental.map_and_batch` to dynamically tune the level of parallelism at runtime.
```

Comment:
@goldiegadde please move this under `tf.data` in the `Bug Fixes and Other Changes` ... this feature is experimental and @tfboyd ran into some issues with using it, so I would prefer if we do not tout it as a major feature

Reply:
Done.